cost/azure/rightsize_managed_disks/README.md

```diff
@@ -0,0 +1,61 @@
+- *Allow/Deny Subscriptions List* - A list of allowed or denied Subscription IDs/names. If empty, no filtering will occur and recommendations will be produced for all subscriptions.
+- *Allow/Deny Regions* - Whether to treat Allow/Deny Regions List parameter as allow or deny list. Has no effect if Allow/Deny Regions List is left empty.
+- *Allow/Deny Regions List* - Filter results by region, either only allowing this list or denying it depending on how the above parameter is set. Leave blank to consider all the regions.
+- *Minimum used disk space percentage to consider a managed disk for downsizing* - The minimum used disk space percentage threshold at which to consider a disk to be 'oversized' and therefore be flagged for downsizing.
+- *Minimum used disk IOPS percentage to consider a managed disk for downsizing* - The minimum used disk IOPS percentage threshold at which to consider a disk to be 'oversized' and therefore be flagged for downsizing.
+- *Aggregation type for determining oversized disk IOPS* - Statistic to use when determining if the disk IOPS is more than the disk actually uses.
+- *Minimum used disk throughput percentage to consider a managed disk for downsizing* - The minimum used disk throughput percentage threshold at which to consider a disk to be 'oversized' and therefore be flagged for downsizing.
+- *Aggregation type for determining oversized disk throughput* - Statistic to use when determining if the disk throughput is more than the disk actually uses.
+- *Lookback period* - How many days back to look at disk IOPS and throughput data. This value cannot be set higher than 90 because Azure does not retain metrics for longer than 90 days.
+
+## Policy Actions
+
+The following policy actions are taken on any resources found to be out of compliance.
+
+- Send an email report
+
+## Prerequisites
+
+This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
+
+- [**Azure Resource Manager Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm#automationadmin_109256743_1124668) (*provider=azure_rm*) which has the following permissions:
+
+  - `Microsoft.Compute/disks/read`
+  - `Microsoft.Compute/virtualMachines/read`
+  - `Microsoft.Insights/metrics/read`
+
+- [**Flexera Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) (*provider=flexera*).
+
+This is only required for the application of the metaparent policy.
```

Comment:
This should be modified to match other policies. Also, we should use this credential to pull the policy name and include that in the title of the incident, similar to our other cost recommendation policies. See Azure Rightsizing Compute for an example:

```# Get applied policy metadata for use later
datasource "ds_applied_policy" do
  request do
    auth $auth_flexera
    host rs_governance_host
    path join(["/api/governance/projects/", rs_project_id, "/applied_policies/", policy_id])
    header "Api-Version", "1.0"
  end
end

Reply:
Thank you @XOmniverse , I applied these changes!